libpod/networking_linux.go

```diff
@@ -262,6 +297,14 @@ func (r *Runtime) setupRootlessNetNS(ctr *Container) error {
 		cmdArgs = append(cmdArgs, "--enable-seccomp")
 	}
 
+	if outboundAddr != nil {
```

Comment:
The default string type of golang is null, which is "", can not use nil in string